app/pathogen/arbovirus/analyze/page.tsx

```diff
@@ -4,61 +4,77 @@ import React, { useCallback, useMemo } from "react";
+          {props.renderIcons()}
+        </div>
         {props.children}
       </CardContent>
     </Card>
   );
 };
 
 export default function ArboAnalyze() {
-  const renderPlotCardContent = useCallback(({ cardConfigurations }: {cardConfigurations: CardConfiguration[]}) => (
-    <>
-      <VisualizationCard title={"Median seroprevalence by WHO region and age group"} height="h-full">
-        <MedianSeroPrevByWHOregionAndAgeGroup />
-      </VisualizationCard>
-      <VisualizationCard title={"Estimate count by WHO region and arbovirus"}>
-        <WHORegionAndArbovirusBar />
-      </VisualizationCard>
-      <VisualizationCard title={"Estimate count by arbovirus & antibody type"}>
-        <AntibodyPathogenBar />
-      </VisualizationCard>
-      <VisualizationCard title={"Median seroprevalence by WHO Region"} height="h-full">
-        <MedianSeroPrevByWHOregion />
-      </VisualizationCard>
-      <VisualizationCard title={"Cumulative estimate count over time by arbovirus"}>
-        <StudyCountOverTime />
-      </VisualizationCard>
-      <VisualizationCard title={"Cumulative estimate count over time by sample frame"}  height="h-full 2xl:h-3/4">
-        <StudyCountOverTimeBySampleFrame/>
-      </VisualizationCard>
-      <VisualizationCard title={"Top ten countries reporting estimates by arbovirus"} height="h-full">
-        <Top10CountriesByPathogenStudyCount />
-      </VisualizationCard>
-    </>
-  ), []);
+  const router = useRouter();
+
+  const renderPlotCardContent = useCallback(({ cardConfigurations }: {cardConfigurations: CardConfiguration[]}) => {
+    const allVisualizationInformationWithHeights = addToVisualizationInformation({
+      [VisualizationId.MEDIAN_SEROPREVALENCE_BY_WHO_REGION_AND_AGE_GROUP]: { height: "h-full" },
+      [VisualizationId.ESTIMATE_COUNT_BY_WHO_REGION_AND_ARBOVIRUS]: { height: undefined },
+      [VisualizationId.ESTIMATE_COUNT_BY_ARBOVIRUS_AND_ANTIBODY_TYPE ]: { height: undefined },
+      [VisualizationId.MEDIAN_SEROPREVALENCE_BY_WHO_REGION]: { height: "h-full" },
+      [VisualizationId.CUMULATIVE_ESTIMATE_COUNT_OVER_TIME_BY_ARBOVIRUS]: { height: undefined },
+      [VisualizationId.CUMULATIVE_ESTIMATE_COUNT_OVER_TIME_BY_SAMPLE_FRAME]: { height: "h-full 2xl:h-3/4" },
+      [VisualizationId.TOP_TEN_COUNTRIES_REPORTING_ESTIMATES_BY_ARBOVIRUS]: { height: "h-full" },
+    })
+
+    return (
+      <>
+        {allVisualizationInformationWithHeights.map((visualizationInformation) => (
+          <VisualizationCard 
+            key={visualizationInformation.id}
+            title={visualizationInformation.displayName}
+            height={visualizationInformation.height}
+            renderIcons={() => 
+              <button
+                onClick={() => router.push(`visualizations?visualization=${visualizationInformation.urlParameter}&referrerRoute=/pathogen/arbovirus/analyze`)}
+                aria-label="See visualization in fullscreen"
+              >
+                <ZoomIn />
```

Comment:
I think the zoom button is really cute because
1. We can make more clickable things on the visualization if we'd like
2. You actually know what's happening if you click on the button as opposed to clicking on the visualization.
3. I think it would be kind of cute to throw a download button or something next to it potentially for when we want to do the images
4. Getting a `onClick` for the entire visualization seems like a pain

Reply:
yEAH i REALLY LIKE THE zoom button, it is more intuitive for sure!